front/app/components/admin/ActionForm/CustomizeErrorMessage/ErrorMessageModal.tsx

```diff
@@ -0,0 +1,132 @@
+
+import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
+import useLocale from 'hooks/useLocale';
+
+import accessDeniedMessages from 'containers/Authentication/steps/AccessDenied/messages';
+
+import QuillMultilocWithLocaleSwitcher from 'components/HookForm/QuillMultilocWithLocaleSwitcher';
+import Modal from 'components/UI/Modal';
+
+import { FormattedMessage } from 'utils/cl-intl';
+import useFormatMessageWithLocale from 'utils/cl-intl/useFormatMessageWithLocale';
+import { handleHookFormSubmissionError } from 'utils/errorUtils';
+
+import messages from './messages';
+
+interface FormValues {
+  access_denied_explanation_multiloc?: Multiloc;
+}
+
+interface Props {
+  opened: boolean;
+  access_denied_explanation_multiloc?: Multiloc;
+  onClose: () => void;
+  onSubmit: (formValues: FormValues) => Promise<void>;
+}
+
+const ErrorMessageModal = ({
+  opened,
+  access_denied_explanation_multiloc,
+  onClose,
+  onSubmit,
+}: Props) => {
+  const [selectedLocale, setSelectedLocale] = useState<SupportedLocale>(
+    useLocale()
+  );
+  const formatMessageWithLocale = useFormatMessageWithLocale();
+  const locales = useAppConfigurationLocales();
+
+  const schema = object({
+    access_denied_explanation_multiloc: object(),
+  });
+
+  const methods = useForm({
+    mode: 'onBlur',
+    defaultValues: { access_denied_explanation_multiloc },
+    resolver: yupResolver(schema),
+  });
+
+  const handleSubmit = async (formValues: FormValues) => {
+    try {
+      await onSubmit(formValues);
+      onClose();
+    } catch (error) {
+      handleHookFormSubmissionError(error, methods.setError);
+    }
+  };
+
+  if (!locales) return null;
+
+  return (
+    <Modal
+      opened={opened}
+      close={onClose}
+      niceHeader={true}
+      header={
+        <Title ml="20px" variant="h3" color="primary">
+          <FormattedMessage {...messages.customizeErrorMessage} />
+        </Title>
+      }
+      closeOnClickOutside={false}
+      width={'550px'}
+    >
+      <Box m="20px">
+        <Text mt="0px">
+          <FormattedMessage {...messages.defaultErrorMessageExplanation} />
+        </Text>
+        <Box display="flex" flexDirection="column" alignItems="center">
+          <Box width="auto">
+            <LocaleSwitcher
+              selectedLocale={selectedLocale}
+              locales={locales}
+              onSelectedLocaleChange={(locale: SupportedLocale) =>
+                setSelectedLocale(locale)
+              }
+            />
+          </Box>
+          <Text fontSize="s">
+            {formatMessageWithLocale(
+              selectedLocale,
+              accessDeniedMessages.youDoNotMeetTheRequirements
+            )}
+          </Text>
+          <Text mb="40px" mt="0px">
+            <FormattedMessage {...messages.customizeErrorMessageExplanation} />
+          </Text>
+        </Box>
+        <FormProvider {...methods}>
+          <form onSubmit={methods.handleSubmit(handleSubmit)}>
```

Comment:
Yeah this is just what the form library that we use, `react-hook-form`, recommends. See for example https://react-hook-form.com/get-started. 

I think we could consider abstracting our forms a bit more, since setting up forms currently still involves a lot of boilerplate (e.g. we always need this `FormProvider` followed directly by the `form` element- this could perhaps be combined into one component).

But yeah I guess out of scope for this PR